bot/engine/src/main/kotlin/engine/config/RAGAnswerHandler.kt

```diff
@@ -156,8 +160,22 @@ object RAGAnswerHandler : AbstractProactiveAnswerHandler {
     private fun rag(botBus: BotBus): RAGResult {
         logger.info { "Call Generative AI Orchestrator - RAG API" }
         with(botBus) {
-
+            // The RAG Story is only handled when RAG and Vector Store configurations are enabled
```

Comment:
According to the implementation it will also work without VectorStore enabled as it will send a null `vectorStoreSetting` in the `RAGQuery` which will default to environment configuration at the Gen AI Orchestrator level and still work if they are correctly defined.
I think we should describe this logic here : 
```suggestion
            // The RAG Story is only handled when RAG and will use Vector Store settings if defined, otherwise it will not send any vector store setting in the RAG Query which will default to Gen AI Orchestrator environment variable vector settings
```